slack_mcp_server.py

```
@@ -213,6 +203,87 @@ def _save_user_cache() -> None:
+    Emails are excluded from the persisted payload to avoid storing PII on disk.
+    """
+    try:
+        sanitized = [{k: v for k, v in u.items() if k != "email"} for u in users]
+        data = {
+            "fetched_at": datetime.now(timezone.utc).isoformat(),
+            "users": sanitized,
+        }
+        with open(REVERSE_USER_CACHE_FILE, 'w') as f:
+            json.dump(data, f, indent=2)
+        log(f"Saved reverse user cache ({len(sanitized)} users)")
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)